commons-eid-bom/pom.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+	<parent>
```

Comment:
That commit is not really required, and just made it easier to propagate a 'man versions:set' to all maven modules (we're using internal releases for the moment with our fixes applied). I will roll it back.